doc/api/os.md

```diff
@@ -33,205 +44,294 @@ defined are described in [OS Constants][].
-       internal: true },
-     { address: '::1',
-       netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
-       family: 'IPv6',
-       mac: '00:00:00:00:00:00',
-       internal: true } ],
-  eth0:
-   [ { address: '192.168.1.108',
-       netmask: '255.255.255.0',
-       family: 'IPv4',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false },
-     { address: 'fe80::a00:27ff:fe4e:66a1',
-       netmask: 'ffff:ffff:ffff:ffff::',
-       family: 'IPv6',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false } ] }
+{
+  lo: [
+    {
+      address: '127.0.0.1',
+      netmask: '255.0.0.0',
+      family: 'IPv4',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    },
+    {
+      address: '::1',
+      netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
+      family: 'IPv6',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    }
+  ],
+  eth0: [
+    {
+      address: '192.168.1.108',
+      netmask: '255.255.255.0',
+      family: 'IPv4',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    },
+    {
+      address: 'fe80::a00:27ff:fe4e:66a1',
+      netmask: 'ffff:ffff:ffff:ffff::',
+      family: 'IPv6',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    }
+  ]
+}
 ```
 
-Note that due to the underlying implementation this will only return network
-interfaces that have been assigned an address.
-
 ## os.platform()
 <!-- YAML
 added: v0.5.0
 -->
 
-Returns the operating system platform. Possible values are `'darwin'`,
-`'freebsd'`, `'linux'`, `'sunos'` or `'win32'`. Returns the value of
-[`process.platform`][].
+The `os.platform()` method returns a string identifying the operating system
+platform as set during compile time of Node.js.
+
+Currently possible values are:
+
+* `'aix'`
+* `'darwin'`
+* `'freebsd'`
+* `'linux'`
+* `'openbsd'`
+* `'sunos'`
+* `'win32'`
+
+Equivalent to [`process.platform`][].
+
+*Note*: The value `'android'` may also be returned if the Node.js is built on
+the Android operating system. However, Android support in Node.js is considered
+to be experimental at this time.
 
 ## os.release()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the operating system release.
+The `os.release()` method returns a string identifying the operating system
+release.
 
 ## os.tmpdir()
 <!-- YAML
 added: v0.9.9
 -->
 
-Returns the operating system's default directory for temporary files.
+The `os.tmpdir()` method returns the operating system's default directory for
+temporary files.
 
 ## os.totalmem()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the total amount of system memory in bytes.
+The `os.totalmem()` method returns the total amount of system memory in bytes
+as an integer.
 
 ## os.type()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the operating system name. For example `'Linux'` on Linux, `'Darwin'`
-on OS X and `'Windows_NT'` on Windows.
+The `os.type()` method returns the a string identifying the operating system
+name as returned by uname(3). For example `'Linux'` on Linux, `'Darwin'` on
```

Comment:
`uname(3)` should have a posix link like other such man page links now have in the docs.